directory/directory_src.go

```diff
@@ -35,7 +36,7 @@ func (s *dirImageSource) GetManifest() ([]byte, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	return m, "", err
+	return m, manifest.GuessMIMEType(m), err
```

Comment:
Wait `types.ImageSource.GetManifest` explicitly says that `'""` means unknown. Either this is wrong and we should still be guessing in `containers/image/image` if the MIME type is empty, or the documentation of `GetManifest` needs updating.